<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrightnessContrastDialog</class>
 <widget class="QDialog" name="BrightnessContrastDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Яркость/контрастность/интенсивность</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icegui.qrc">
    <normaloff>:/dvg/icegui/images/contrast.png</normaloff>:/dvg/icegui/images/contrast.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="brightnessL">
       <property name="text">
        <string>Яркость:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="brightnessSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="brightnessSB">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="contrastL">
       <property name="text">
        <string>Контрастность:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="contrastSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="contrastSB">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="intensityL">
       <property name="text">
        <string>Интенсивность:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="intensitySlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="intensitySB">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icegui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>contrastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrastSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrastSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrastSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightnessSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightnessSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>BrightnessContrastDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>BrightnessContrastDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intensitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>intensitySB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intensitySB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>intensitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightnessContrastDialog</receiver>
   <slot>brightnessSlider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightnessContrastDialog</receiver>
   <slot>contrastSlider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intensitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightnessContrastDialog</receiver>
   <slot>intensitySlider_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>contrastSlider_valueChanged(int)</slot>
  <slot>brightnessSlider_valueChanged(int)</slot>
  <slot>intensitySlider_valueChanged(int)</slot>
 </slots>
</ui>
